src/main/resources/env-map.yml

```diff
@@ -1,14 +1,15 @@
 env_vars:
-  BASE_URL:                    '.[][] | select(.name == "app-catalog")               | .credentials.adminusers_url'
-  SELFSERVICE_URL:             '.[][] | select(.name == "app-catalog")               | .credentials.selfservice_url'
-  SUPPORT_URL:                 '.[][] | select(.name == "app-catalog")               | .credentials.card_frontend_support_url'
-  DB_HOST:                     '.[][] | select(.name == "adminusers-db")             | .credentials.host'
-  DB_NAME:                     '.[][] | select(.name == "adminusers-db")             | .credentials.name'
-  DB_PASSWORD:                 '.[][] | select(.name == "adminusers-db")             | .credentials.password'
-  DB_USER:                     '.[][] | select(.name == "adminusers-db")             | .credentials.username'
-  DB_SSL_OPTION:               '.[][] | select(.name == "adminusers-db")             | .credentials.ssl_option // "ssl=true"'
-  NOTIFY_API_KEY:              '.[][] | select(.name == "adminusers-secret-service") | .credentials.notify_api_key'
-  NOTIFY_DIRECT_DEBIT_API_KEY: '.[][] | select(.name == "adminusers-secret-service") | .credentials.notify_direct_debit_api_key'
-  SENTRY_DSN:                  '.[][] | select(.name == "adminusers-secret-service") | .credentials.sentry_dsn'
-  METRICS_HOST:                '.[][] | select(.name == "app-catalog")               | .credentials.carbon_relay_route'
-  METRICS_PORT:                '.[][] | select(.name == "app-catalog")               | .credentials.carbon_relay_port'
+  BASE_URL:                                '.[][] | select(.name == "app-catalog")               | .credentials.adminusers_url'
+  SELFSERVICE_URL:                         '.[][] | select(.name == "app-catalog")               | .credentials.selfservice_url'
+  SUPPORT_URL:                             '.[][] | select(.name == "app-catalog")               | .credentials.card_frontend_support_url'
+  DB_HOST:                                 '.[][] | select(.name == "adminusers-db")             | .credentials.host'
+  DB_NAME:                                 '.[][] | select(.name == "adminusers-db")             | .credentials.name'
+  DB_PASSWORD:                             '.[][] | select(.name == "adminusers-db")             | .credentials.password'
+  DB_USER:                                 '.[][] | select(.name == "adminusers-db")             | .credentials.username'
+  DB_SSL_OPTION:                           '.[][] | select(.name == "adminusers-db")             | .credentials.ssl_option // "ssl=true"'
+  NOTIFY_API_KEY:                          '.[][] | select(.name == "adminusers-secret-service") | .credentials.notify_api_key'
+  NOTIFY_INVITE_SERVICE_EMAIL_TEMPLATE_ID: '.[][] | select(.name == "adminusers-secret-service") | .credentials.notify_invite_service_email_template_id'
```

Comment:
do these template ids need to change per environment or is it always the same? If so it could go into the manifest directly.

Reply:
largely the same but there are some which swap around based on environment